lms/envs/common.py

```diff
@@ -1596,6 +1596,13 @@
     'WHAT_IS_VERIFIED_CERT': 'verified-certificate',
 }
 
+################# Mobile URLS ##########################
+
+# These are URLs to the app store for mobile.
+MOBILE_STORE_URLS = {
+    'apple': '#',
+    'google': '#'
```

Comment:
We actually have the google play url https://play.google.com/store/apps/details?id=org.edx.mobile